tasks/qunit.js

```diff
@@ -22,7 +22,15 @@ var Promise = global.Promise;
 // Shared functions
 
 // Allow an error message to retain its color when split across multiple lines.
-function formatMessage (str) {
+function formatMessage (message) {
+  var str = String(message);
+  if (typeof message === 'object' && /^\[object .*\]$/.test(str)) {
```

Comment:
This is supposed to be a rudimentary “does this object have a useful `toString()` method” check (`message.toString === Object.prototype.toString` didn’t work). Maybe `str === '[object Object]'` would also be enough.